vibe_code_apps_20/examples/workspace_tools/AGENT_CONTEXT.md

```
@@ -0,0 +1,131 @@
+  cascade (never invent).
+
+---
+
+## Where to go deeper (optional)
+
+| Doc | When |
+| --- | --- |
+| [`TWIN_DIAL_PLAYBOOK.md`](TWIN_DIAL_PLAYBOOK.md) | Ordered phases + practice trajectory |
+| Skill `wattlab-twin-calibrate-dial` | Agent skill frontmatter + same phases |
+| [`AGENT_TOOLS.md`](AGENT_TOOLS.md) | `/data/tools` campaign scripts vs product CLIs |
+| [`BUG_REPORT_ECM_SPREADSHEET_VS_EPLUS.md`](BUG_REPORT_ECM_SPREADSHEET_VS_EPLUS.md) | BUG-ECM-* / ENH-ECM-* register |
```

markdownlint-cli2 (0.23.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)